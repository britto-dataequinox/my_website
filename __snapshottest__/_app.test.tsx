import React from 'react';
import { render } from '@testing-library/react';
import MyApp from '@/pages/_app';

jest.mock('next/router', () => ({
  events: {
    on: jest.fn(),
    off: jest.fn(),
  },
  useRouter: jest.fn(),
}));

describe('MyApp', () => {
  test('my app snap', () => {
    const { asFragment } = render(
      <MyApp
        Component={() => <div data-testid="component">Test Component</div>}
        pageProps={{}}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
