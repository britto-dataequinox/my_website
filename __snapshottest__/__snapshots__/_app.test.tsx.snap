// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp my app snap 1`] = `
<DocumentFragment>
  <div
    data-testid="component"
  >
    Test Component
  </div>
</DocumentFragment>
`;
